### PR TITLE
stub for email signature removal + test

### PR DESCRIPTION
**Pull request by martinraison**
@stkem ended up not using the complete email, actually (I don't see any way to properly instantiate a Java Message object from a string, so for now we're just testing the regex, not the email body extraction)

### CODE REVIEW

**Comment by stkem** [comment]
lgtm

File: kifi-backend/modules/eliza/app/com/keepit/eliza/mail/MailMessageReceiverPlugin.scala [hunk 1]
```diff
@@ -108,12 +108,20 @@ class MailDiscussionMessageParser @Inject() (
 
   def getInfo(message: Message): Option[MailNotificationReply] = {
     getPublicId(message) map { publicId =>
-      MailNotificationReply(getTimestamp(message), getText(message).map(s => (new Regex(raw"\n[^\n]*((<[\s\S]+@[\s\S]+>)|(\([\s\S]+@[\s\S]+\)))[^\n]*:")).split(s)(0).trim), publicId)
+      MailNotificationReply(getTimestamp(message), getText(message).map{ s => MailDiscussionMessageParser.extractMessage(s) }, publicId)
```
Comment (2is10) [inline kifi-backend/modules/eliza/app/com/keepit/eliza/mail/MailMessageReceiverPlugin.scala]: you can shorten the `.map` to `.map(MailDiscussionMessageParser.extractMessage)`

**Comment by 2is10** [comment]
lgtm. thanks for adding tests!

File: kifi-backend/modules/eliza/app/com/keepit/eliza/mail/MailMessageReceiverPlugin.scala (hunk 1 above)
Comment (martinraison) [inline kifi-backend/modules/eliza/app/com/keepit/eliza/mail/MailMessageReceiverPlugin.scala]: good catch

**Pull request merged**